### DISCUSSION TITLE
Why the app is connecting to host on every startup?

### DISCUSSION THREAD

**Discussion by chromer030** [damionx7/Zettel-Notes-Documentation#97] [General]
![Screenshot_20240625-091942_mpv_1](https://github.com/damionx7/Zettel-Notes-Documentation/assets/13361552/ffb0431c-b4fa-482c-ac4e-45fed3daecfe)

**Comment by damionx7** [comment]
Hey. On startup, CSL json file (needed for bibtex) is checked for any update (hosted on znotes.thedoc.eu.org). Everything hosted there is open source in this repository.
